### ISSUE TITLE
Optionally Use block decorations for charts

### ISSUE THREAD

**Issue by jasongilman** [jasongilman/proto-repl-charts#6]
See http://blog.atom.io/2016/02/03/introducing-block-decorations.html

It should be possible to display a chart inside of a block decoration so that the clojure file containing the chart code could display the chart. Another idea is to put decorations inside the REPL itself. However it would scroll off eventually. If it goes in the same file then it could be replaced as the code is re-executed.

**Comment by whilo** [comment]
Yes, this was awesome. I am doing scientific computing and statistics and always wanted to have that in lighttable. Gorilla REPL was nice, but I think proto-REPL would be perfect if it could show plots inline (probably foldable). So :+1:
